osmaxx-py/excerptconverter/converter_helper.py

```diff
@@ -7,6 +6,42 @@
 from osmaxx.excerptexport import models
 
 
+def module_converter_configuration(name, export_formats, export_options):
+    """
+    :param export_formats example:
+        {
+            'txt': {
+                'name': 'Text',
+                'file_extension': 'txt',
+                'mime_type': 'text/plain'
+            },
+            'markdown': {
+                'name': 'Markdown',
+                'file_extension': 'md',
+                'mime_type': 'text/markdown'
+            }
+        }
+    :param export_options example:
+        {
+            'image_resolution': {
+                'label': 'Resolution',
+                'type': 'number',
+                'default': '500'
+            },
+            'quality': {
+                'label': 'Quality',
+                'type': 'number',
+                'default': '10'
+            }
+        }
+    """
+    return {
```

Comment:
I can see the use of this, I guess. Let's just leave it as it is, but making this into a config (ie. settings) using something like an [ConfigParser](https://docs.python.org/3.4/library/configparser.html) is IMHO the better approach, if it really is a configuration (it does look like it ;) )